_topic_map.yml

```diff
@@ -71,6 +71,21 @@ Topics:
 - Name: Migrating to Operator SDK v0.1.0
   File: migrating-to-osdk-v0-1-0
 ---
+Name: Working with nodes
+Dir: nodes
+Distros: openshift-*
+Topics:
+- Name: Freeing node resources using garbage collection
+  File: nodes-nodes-garbage-collection
+- Name: Allocating resources for nodes
+  File: nodes-nodes-resources-configuring
+- Name: Adversiting hidden resources for nodes
```

Comment:
@mburke5678 typo. Advertising.

Reply:
My plan is to punch everything into 4.0, then go back and look more closely at each PR before turning over for reviews.